src/app/subcategory/components/createSubcategoryForm/createSubcategoryForm.tsx

```diff
@@ -84,12 +83,11 @@ export const CreateSubcategoryForm = ({
 
     if (currentCategoryId) {
       const response = await createSubcategory(currentCategoryId, data);
-      const isError = !response || ('error' in response && !!response.error);
-      if (isError) {
-        alert(response?.error || 'Error creating list');
+      if (response.error) {
+        alert(response.error || 'Error creating list');
```

Comment:
now I see that you have already added validation, so could you please change these alerts to messages as in other parts of app (it was temporary decision)